### PR TITLE
Adding new generated membership cards

### PR DESCRIPTION
**Pull request by is-isa**
New membership cards generated after the Yosemite event. 
Smaller corrections to the email message

### CODE REVIEW

File: members/Nicolas_Tschumy.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Nicolas Tschumy
+---
+
+Status: Phd student
+
+Expiration date: 2019.01.01
+
+Validity: <font color="red"> Not valid</font>
```
Comment (ibogomolov) [inline members/Nicolas_Tschumy.md:10]: The status should be valid. Looks like Nicolas Tschumy paid:

16/12/2018 22:28:41 | Nicolas |   | Tschumy | Phd student |   | https://drive.google.com/open?id=1r75LNFhguWTIm174SHcOKRU880oXxjh_ | nicolas.tschumy@unil.ch | TRUE | TRUE |   |   |   |   |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

Do you have an idea why it's not vaild?

File: members/Victor_Brasch.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Victor Brasch
+---
+
+Status: External
+
+Expiration date: 2019.01.01
+
+Validity: <font color="green"> Verified</font>
```
Comment (ibogomolov) [inline members/Victor_Brasch.md:10]: Here validity check was done correctly.

File: members/Nicolas_Tschumy.md (hunk 1 above)
Comment (is-isa) [reply members/Nicolas_Tschumy.md:10]: This is totally normal, I added him to the list of people who paid after having run the script

File: members/Victor_Brasch.md (hunk 2 above)
Comment (is-isa) [reply members/Victor_Brasch.md:10]: What do you mean?

**Pull request merged**